presto-hive/pom.xml

```diff
@@ -148,6 +148,12 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.jodd</groupId>
+            <artifactId>jodd-core</artifactId>
```

Comment:
Can we do this with joda-time, rather than introducing yet another datetime library?